src/android/notification/Options.java

```diff
@@ -380,6 +380,11 @@ int getSmallIcon() {
         String icon = options.optString("smallIcon", DEFAULT_ICON);
         int resId   = assets.getResId(icon);
 
+        if (resId == 0) {
```

Comment:
Isn't this being explicitly removed in https://github.com/OutSystems/cordova-plugin-local-notifications/pull/9/commits/b8f358ebb9ed76a1ddd61892a66dee1777042bfc?

Reply:
Maybe the context is important.

Reply:
Very nice observation.
However, i would say that this way of doing things was more related to a design opinion on the _commiter_ side, as i inferred in the commit message
> The application icon is set always and it is not able to select the resId = android.R.drawable.ic_popup_reminder. This is always set (API 1) and not showing ugly as app icon (notification small icon should be a silhouette)

Not related to a contextual bug.